RELEASE-NOTES.md

```diff
@@ -18,11 +18,13 @@
 ## Backward incompatibility
 
 ## Deprecations
+* Added deprecation warning in the description of `snow spcs service status` and `snow spcs image-repository list-tags`.
 
 ## New additions
-
+* Added `snow spcs service list-instances`, `snow spcs service list-containers` and `snow spcs service list-roles` commands, which support fetching information about all instances/containers/service roles in a service.
 
 ## Fixes and improvements
+* `snow spcs service set` now supports `--eai-name` to update external access integrations for a service.
```

Comment:
Probably also mentioning that we improve the `snow spcs service list-images` result by including enriched info like tag and sha

Reply:
Good point! I will update.